diff --git a/node_modules/@nodeml/audio/lib/index.d.ts b/node_modules/@nodeml/audio/lib/index.d.ts
index ff28ea9..e700e3f 100644
--- a/node_modules/@nodeml/audio/lib/index.d.ts
+++ b/node_modules/@nodeml/audio/lib/index.d.ts
@@ -25,15 +25,15 @@ export namespace stream {
     
     
     namespace formats {
-        declare const float32: 0x00000001;
-        declare const int32: 0x00000002;
-        declare const int24: 0x00000004;
-        declare const int16: 0x00000008;
-        declare const int8: 0x00000010;
-        declare const uint8: 0x00000020;
+         const float32: 0x00000001
+         const int32: 0x00000002
+         const int24: 0x00000004
+         const int16: 0x00000008
+         const int8: 0x00000010
+         const uint8: 0x00000020
     }
     
-    type StreamFormats = keyof formats | undefined
+    type StreamFormats = typeof formats[keyof typeof formats] | undefined
     
     interface IFormatsToArrays {
         [formats.float32]: Float32Array;
@@ -71,7 +71,7 @@ export namespace stream {
         output: IStreamParams<OutputFormat>
     }
     
-    declare class Stream<OutputFormat extends StreamFormats> {
+    class Stream<OutputFormat extends StreamFormats> {
         start: () => void;
     
         stop: () => void;
@@ -81,13 +81,13 @@ export namespace stream {
         write: (data: OutputFormat extends undefined ? undefined : IFormatsToArrays[OutputFormat]) => boolean;
     }
     
-    declare function open<InputFormat extends StreamFormats = undefined, OutputFormat extends StreamFormats = undefined>(config: StreamConfig<InputFormat,OutputFormat>, sampleRate: number, framesPerBuffer: number): Stream<OutputFormat>;
+    function open<InputFormat extends StreamFormats = undefined, OutputFormat extends StreamFormats = undefined>(config: StreamConfig<InputFormat,OutputFormat>, sampleRate: number, framesPerBuffer: number): Stream<OutputFormat>;
     
-    declare function getHosts(): IPortAudioHost[]
+    function getHosts(): IPortAudioHost[]
     
-    declare function getHostDevices(hostId: number): IPortAudioDevice[]
+    function getHostDevices(hostId: number): IPortAudioDevice[]
     
-    declare function getDefaultHostIndex(): number
+    function getDefaultHostIndex(): number
 }
 
 // export namespace wav {
